src/main/resources/admin/tools/content-type-manager/components/structure/component-form-definition.html

```diff
@@ -0,0 +1,172 @@
+<div th:fragment="component-form-definition" th:remove="tag">
+  <template>
+    <v-card>
+      <div
+        class="row pointer"
+        @click="showConfig = !showConfig"
+      >
+        <v-col cols="12">
+          <v-card-title>Form Definition</v-card-title>
+        </v-col>
+      </div>
+
+      <transition name="fade">
+        <v-card-text v-show="showConfig">
+          <v-divider class="mb-5"></v-divider>
+
+          <template
+            v-if="Object.keys(map).length > 0"
+            v-for="(element, index) in remainingElements"
+          >
+            <component
+              v-if="map[element.name] !== undefined"
+              :is="map[element.name].componentName"
+              v-bind="{ 
+                path: depthPath(index),
+                initialValue: element.initialValue || '',
+                field: map[element.name].field,
+                items: map[element.name].items,
+                rules: map[element.name].rules,
+                i18n: map[element.name].i18n
+              }"
+            ></component>
+            <component
+              v-else-if="element.name && element.name !== 'form'"
+              :is="'component-form-definition-' + element.name"
+              v-bind="{ path: depthPath(index), inputType: fileName }"
+            ></component>
+          </template>
+
+          <component
+            :is="'component-input-form-definition-allow-child-content-types'"
+            v-bind="{ path, inputType: fileName }"
+          ></component>
+
+          <component
+            :is="'component-input-form-definition-x-datas'"
+            v-bind="{ path, inputType: fileName }"
+          ></component>
+        </v-card-text>
+      </transition>
+    </v-card>
+  </template>
+
+  <script th:inline="javascript">
+    DOM = document.getElementsByTagName("template")[0];
```

Comment:
Why do you need template tag if its going to be removed on script execution? 
`<th:block>` or `th:remove="tag"` should do the trick?